docs/configuration.rst

```diff
@@ -80,6 +80,23 @@ And give them permissions by schema :
 
 You can also create groups, etc. See PostgreSQL documentation.
 
+Activate SSL / HTTPS
+--------------------
+
+If you want to activate https, you need firstly to change custom.py and add :
```

Comment:
I would say "To activate HTTPS… " instead of "If you want to activate…" because it is mandatory in many case
```suggestion
To activate https, you need firstly to change custom.py and add :
```